src/brsTypes/BrsType.ts

```diff
@@ -123,13 +125,13 @@ export function getBrsValueFromFieldType(type: string, value?: string): BrsType
         case "float":
             returnValue = value ? new Float(Number.parseFloat(value)) : new Float(0);
             break;
-        case "roArray":
```

Comment:
We're calling `type.toLowerCase()` on [line 113](https://github.com/sjbarag/brs/pull/432/files#diff-cef4e231ed0b97ed2e188d7534bed1d2R113), so this should be lowercase instead of camelcase.